src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java

```diff
@@ -542,6 +559,11 @@ void getAllProjectManagementEntries_whenNoStepEntriesFound_thenReturnsResultList
         assertThat(result).isEmpty();
     }
 
+    private Duration getDurationFromTimeString(String timeString) {
```

Comment:
hier entfernen und das Util verwenden

Reply:
muss hier bleiben, weil das Util gemocked verwendet wird (an anderer Stelle in der Testklasse)